### ISSUE TITLE
fix deprecation warnings (>=Symfony 4.3)

### ISSUE THREAD

**Issue by PWalkow** [welpdev/mailchimp-bundle#32]
The "Symfony\Component\Config\Definition\Builder\TreeBuilder::root()" method called for the "welp_mailchimp" configuration is deprecated since Symfony 4.3, pass the root name to the constructor instead.

very similiar to #25 

example of fix in doctrine-bundle
```
$treeBuilder = new TreeBuilder('doctrine');

if (method_exists($treeBuilder, 'getRootNode')) {
    $rootNode = $treeBuilder->getRootNode();
} else {
    // BC layer for symfony/config 4.1 and older
   $rootNode = $treeBuilder->root('doctrine');
}
```

**Comment by Nightbr** [comment]
Hey, thanks for the report

Feel free to open a Pull Request for this 👌

**Comment by PWalkow** [comment]
@Nightbr , sure thing!

**Comment by Nightbr** [comment]
Fixed in [`1.3.2`](https://github.com/welpdev/mailchimp-bundle/releases/tag/1.3.2)
